src/VecSim/algorithms/brute_force/vector_block.h

```diff
@@ -34,6 +50,11 @@ struct VectorBlock {
         this->members[index] = member;
     }
 
+    std::vector<std::pair<float, labelType>> ComputeScores(DISTFUNC<float> DistFunc,
+                                                           const void *queryBlob);
+
+    void heapBasedSearch(const std::vector<std::pair<float, labelType>> &scores, float &upperBound,
```

Comment:
should be a part of a vector block?

Reply:
It was designed that way to allow consolidation with the original `TopK` that also uses heap search

Reply:
It was designed that way to allow consolidation with the original `TopK` that also uses heap search

Reply:
removed